elements/pfe-switch/src/pfe-switch.scss

```diff
@@ -0,0 +1,219 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: switch;
+
+// PFE Config
+$font-size:     16px;
+$line-height:   24px;
+
+// Setting all toggle sizing, will be converted to em in styles
+$toggle-width:        42px;
+$toggle-height:       $line-height;
+$toggle-knob-diameter:   14px;
+$toggle-knob-offset:  5px;
+
+///
+/// Convert px to em
+///
+/// @param $value {length}  Value to convert with px suffix
+/// @param $base-size {length} Optional, font size to use to calculate em
+/// @return {length} Em version of input
+@function px-to-em($value) {
+  @return $value / $font-size * 1em;
+}
+
+//sass-lint:disable indentation
+// Variables based on PF Next Switch
+$variables: (
+  toggle-knob--default__BackgroundColor: #fff,
+  toggle-knob--disabled__Color: #d2d2d2,
+  toggle-knob__BoxShadow: '0 0.0625rem 0.0625rem 0rem rgba(3, 3, 3, 0.05), 0 0.25rem 0.5rem 0.25rem rgba(3, 3, 3, 0.06)',
+
+  toggle--default__BackgroundColor: #d2d2d2,
+  toggle--checked__BackgroundColor: #06c,
+  toggle--disabled__BackgroundColor: #737679,
+
+  label--default__Color: #151515,
+  label--checked__Color: #737679,
+  label--disabled__Color: #737679,
+);
+//sass-lint:enable indentation
+
+// #{pfe-local(paddingTop)}
+
+:host {
+  @include pfe-set-local($variables);
+  display: inline-block;
+  margin: 0 1.5em 0 0;
+  overflow: hidden;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+.pfe-switch {
+  position: relative;
+}
+
+::slotted(input) {
+  @include pfe-screen-reader-only;
+}
+
+::slotted(.sr-only),
+.sr-only {
```

Comment:
There is an `%sr-only` extend defined that you could use.

Reply:
In this case the usual clip method would make my component unusable, using negative text-indent achieves my purpose with less styles and wouldn't require an extra tag.

Reply:
oh wait, sorry you're looking at older code. In one case I def need to use that extend 😬